qt/aqt/table.py

```diff
@@ -149,11 +148,8 @@ def invert_selection(self) -> None:
     def select_single_card(self, card_id: CardId) -> None:
         """Try to set the selection to the item corresponding to the given card."""
         self.clear_selection()
-        if self.is_notes_mode():
-            self._view.selectRow(0)
-        else:
-            if (row := self._model.get_card_row(card_id)) is not None:
-                self._view.selectRow(row)
+        if (row := self._model.get_card_row(card_id)) is not None:
+            self._view.selectRow(row)
 
```

Comment:
Thanks for the fixes in my code. This one I reverted and fixed `get_item_from_card_id()` instead so the method does as it says it does, even though it's not actually useful with the current implementation.